CHANGELOG.md

```diff
@@ -789,12 +794,12 @@ If these files are supposed to not exist, please update your PR body to include
 
   ```js
   // In danger 0.16.0:
-  const fullDiff = danger.git.diffForFile('foo.js')
-  const addedLines = danger.git.diffForFile('foo.js', ['add'])
-  const removedLines = danger.git.diffForFile('foo.js', ['del'])
+  const fullDiff = danger.git.diffForFile("foo.js")
```

Comment:
I did not add this changes, I think the hooks that the project has are causing this.

Reply:
cool 👍